app/decorators/manageiq/providers/ibm_terraform/configuration_manager/configured_system_decorator.rb

```diff
@@ -5,13 +5,13 @@ class ManageIQ::Providers::IbmTerraform::ConfigurationManager::ConfiguredSystemD
     "Google Cloud"                 => "gce",
     "Huawei Cloud"                 => "huawei",
     "IBM"                          => "ibm",
-    "IBM Cloud Kubernetes Service" => "ibm",
+    "IBM Cloud Kubernetes Service" => "ibm_cloud",
     "IBM Cloud Private"            => "ibm",
```

Comment:
Should IBM Cloud Private use the "ibm_cloud" or "ibm" logo?

Reply:
It looks like they use the ibm_cloud logo
![image](https://user-images.githubusercontent.com/12851112/196964760-91f273ea-f44a-41b6-8a4c-7339329bb9fb.png)